src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/enumerations/ReleaseDateType.kt

```diff
@@ -1,28 +1,28 @@
 package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.enumerations
 
-enum class ReleaseDateType {
-  CRD,
-  LED,
-  SED,
-  NPD,
-  ARD,
-  TUSED,
-  PED,
-  SLED,
-  HDCED,
-  NCRD,
-  ETD,
-  MTD,
-  LTD,
-  DPRRD,
-  PRRD,
-  ESED, // Effective sentence end date
-  ERSED,
-  TERSED,
-  APD,
-  HDCAD,
-  None,
-  Tariff,
-  ROTL,
-  HDCED4PLUS,
+enum class ReleaseDateType(val fullName: String) {
+  CRD("Conditional release date"),
+  LED("Licence expiry date"),
+  SED("Sentence expiry date"),
+  NPD("Non-parole date"),
+  ARD("Automatic release date"),
+  TUSED("Top up supervision expiry date"),
+  PED("Parole eligibility date"),
+  SLED("Sentence and licence expiry date"),
+  HDCED("Home detention curfew release date"),
```

Comment:
It's normally `Home detention curfew eligibility date`
Do we have it as something else somewhere?

Reply:
Yes I copied from FULL_STRING_LOOKUP in the UI but it's Home detention curfew eligibility date elsewhere so maybe FULL_STRING_LOOKUP is wrong?